definitions/variable/technology/electricity-operation.yaml

```
@@ -12,360 +12,71 @@
-      this amount is the maximum available for the grid operator considering the commitment
-      decisions taken by the model
-    unit: MWh
-- Reserve|Electricity|Frequency Containment|Oil:
-    description: Frequency Containment Reserve available for Oil power plants; this
-      amount is the maximum available for the grid operator considering the commitment
-      decisions taken by the model
-    unit: MWh
-- Reserve|Electricity|Frequency Containment|Hydro|Reservoir:
-    description: Frequency Containment Reserve available for "simple Hydro reservoir
-      " units; this amount is the maximum available for the grid operator considering
-      the commitment decisions taken by the model
-    unit: MWh
-- Reserve|Electricity|Frequency Containment|Hydro|Pumped Storage:
-    description: Frequency Containment Reserve available for "Pumped Storage " units;
-      this amount is the maximum available for the grid operator considering the commitment
-      decisions taken by the model
-    unit: MWh
-- Reserve|Electricity|Frequency Containment|Hydro|Run of River:
-    description: Frequency Containment Reserve available for "Run of River" units;
-      this amount is the maximum available for the grid operator considering the commitment
-      decisions taken by the model
-    unit: MWh
-- Reserve|Electricity|Frequency Containment|Hydro|Ocean:
-    description: Frequency Containment Reserve available for Ocean Power plants; this
-      amount is the maximum available for the grid operator considering the commitment
-      decisions taken by the model
-    unit: MWh
-- Reserve|Electricity|Frequency Containment|Solar:
-    description: Frequency Containment Reserve available for Photovoltaic power plants;
-      this amount is the maximum available for the grid operator considering the commitment
-      decisions taken by the model
-    unit: MWh
-- Reserve|Electricity|Frequency Containment|Wind|OnShore:
-    description: Frequency Containment Reserve available for  Wind Onshore power plants;
-      this amount is the maximum available for the grid operator considering the commitment
-      decisions taken by the model
-    unit: MWh
-- Reserve|Electricity|Frequency Containment|Wind|OffShore:
-    description: Frequency Containment Reserve available for  Wind Available forfShore
-      power plants; this amount is the maximum available for the grid operator considering
-      the commitment decisions taken by the model
-    unit: MWh
-- Reserve|Electricity|Frequency Containment|Energy Storage System:
-    description: Frequency Containment Reserve available for energy storage systems,
-      excl. hydro storage; this amount is the maximum available for the grid operator
-      considering the commitment decisions taken by the model
-    unit: MWh
+
+- Reserve|Electricity|Automatic Frequency Restoration|<Fuel>:
+    description: Automatic Frequency Restoration Reserve available for <this fuel>
```

Please note the new usage of the tags in the attributes.
```suggestion
    description: Automatic Frequency Restoration Reserve available for <Fuel>
```